prime-router/src/main/kotlin/metadata/LivdLookup.kt

```diff
@@ -0,0 +1,218 @@
+package gov.cdc.prime.router.metadata
+
+import gov.cdc.prime.router.Metadata
+
+// this will be used by the LivdMappers and the CustomFHIRFunction
+// /need to move most of LivdMappers in here
+// will need to move the tests to look at this as well
+// keep the library the way it was then use the context to use the library to pass in custom functions
+object LivdLookup {
+    /**
```

Comment:
Incomplete kdoc. Missing description. Program preference is to also list the arguments as part of the description.